Utils/trigger_content_nightly_build.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 
-_branch=$1
-_circle_token=$2
+_circle_token=$1
```

Comment:
why did you change the order of the args?

Reply:
I need to check if the `_branch` argument was given or not. Since we pass the value anonymously, if the order was as before and the user did not give a branch `_branch` would have the API KEY value.
Another solution was to add a check for the count of arguments `$#`.